akka-actor/src/main/scala/akka/japi/function/Function.scala

```diff
@@ -15,6 +15,18 @@ trait Function[-T, +R] extends java.io.Serializable {
   def apply(param: T): R
 }
 
+object Function extends java.io.Serializable {
+
+  private final val constFunc: Function[Any, Any] = new Function[Any, Any] {
+    override def apply(param: Any): Any = param
+  }
+
+  /**
+   * A method that returns its input value.
+   */
+  def identity[T](): Function[T, T] = constFunc.asInstanceOf[Function[T, T]]
```

Comment:
this will not be available for the Scala 2.11 artifacts (trait+object) but that is ok (dropping 2.11 support in Akka 2.6 anyway)